livekit-plugins/livekit-plugins-silero/livekit/plugins/silero/vad.py

```diff
@@ -66,6 +67,7 @@ def load(
         activation_threshold: float = 0.5,
         sample_rate: Literal[8000, 16000] = 16000,
         force_cpu: bool = True,
+        onnx_file_path: NotGivenOr[Path] = NOT_GIVEN,
```

Comment:
I think we should support str file path as well?
```suggestion
        onnx_file_path: NotGivenOr[Path | str] = NOT_GIVEN,
```

Reply:
Allowed `str` to be passed as well